### PR TITLE
Simple version Swift

### PR DESCRIPTION
**Pull request by damuellen**

### CODE REVIEW

**Review by mimi89999 — APPROVED** [review]

File: simple.swift [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  var counts = [String:Int]()
+
+  for word in words {
+    let canon = word.lowercased()
+    counts[canon, default: 0] += 1
+  }
+
+  let ordered = counts.sorted { $0.1 > $1.1 }
+
+  ordered.forEach { entry in
```
Comment (mimi89999) [inline simple.swift:14]: Why not `for entry in ordered {`?

File: simple.swift [hunk 2]
```diff
@@ -0,0 +1,21 @@
+import Foundation
+
+let input = FileHandle.standardInput
+
+if let text = String(data: input.readDataToEndOfFile(), encoding: .utf8) {
```
Comment (vojtamolda) [inline simple.swift]: I think this line violates the constraint mentioned the _Problem Statements and Constrains_ section of the [blog post](https://benhoyt.com/writings/count-words/). Solutions aren't supposed to load the entire file into memory. Maximum allowed is a 64KB chunk.

File: simple.swift (hunk 1 above)
Comment (damuellen) [reply simple.swift:14]: I like to use forEach when nothing is mutated in the loop.

File: simple.swift (hunk 2 above)
Comment (damuellen) [reply simple.swift]: I just skimmed the blog post, changed the code accordingly.

**Comment by damuellen** [comment]
Made a mistake.

**Comment by mimi89999** [comment]
@damuellen You can squash your commits and reuse the same PR.

**Comment by mimi89999** [comment]
I guess that you wanted to write:
```
var counts = [String:Int]()

while let line = readLine() {
  let words = line.split { $0.isWhitespace }

  for word in words {
    let canon = word.lowercased()
    counts[canon, default: 0] += 1
  }
}

let ordered = counts.sorted { $0.1 > $1.1 }

ordered.forEach { entry in
  print(entry.key, entry.value.description)
}
```

File: simple.swift (hunk 2 above)
Comment (vojtamolda) [reply simple.swift]: 👍

**Pull request closed without merging**